### ISSUE TITLE
GRCh38 vs hg19

### ISSUE THREAD

**Issue by Miaoyuanyuan777** [FunctionLab/ExPecto#31]
Hi~Dr, thank you for your package.

I would like to ask, beluga model is pre-trained with hg19, while the scRNA-seq data is mapped to GRCh38 genome version, so is it effective and appropriate to use beluga to predict the scRNA-seq data?
I want to use beluga as a feature extractor, but I don't know if I can directly use the model that have provided.

Thank you very much ! best wishes !

**Comment by jzthree** [comment]
Yes you should have no problem applying Beluga it to hg38 sequences. but we do have a newer and much improved model Sei that you might want to use instead of Belgua https://github.com/FunctionLab/sei-framework/

**Comment by Miaoyuanyuan777** [comment]
Wow! That great! I will try the improved model Sei.
Thanks for your reply!
Wish you a happy Mid-Autumn festival!
